examples/client/client.c

```diff
@@ -2110,6 +2162,11 @@ THREAD_RETURN WOLFSSL_THREAD client_test(void* args)
             #endif /* HAVE_TRUSTED_CA */
                 break;
 
+            case '6' :
+                nonBlocking = 1;
```

Comment:
As written, WANT_WRITE simulation implies non-blocking mode. Should I require the user to specify -N, or leave it like this, and turn on non-blocking mode automatically if the specify -6?

Reply:
Automatically turning on an implied requirement is ok and simplifies help messages in this case.